Tasks/Xcode/xcode.ts

```diff
@@ -178,14 +179,27 @@ async function run() {
         else if (signingOption === 'manual') {
             xcode_codeSignStyle = 'CODE_SIGN_STYLE=Manual';
 
-            let signIdentity: string = tl.getInput('signingIdentity');
+            const signIdentity: string = tl.getInput('signingIdentity');
             if (signIdentity) {
                 xcode_codeSignIdentity = 'CODE_SIGN_IDENTITY=' + signIdentity;
             }
 
             let provProfileUUID: string = tl.getInput('provisioningProfileUuid');
-            if (provProfileUUID) {
-                xcode_provProfile = 'PROVISIONING_PROFILE=' + provProfileUUID;
+            let provProfileName: string = tl.getInput('provisioningProfileName');
+
+            // If either value is set, override both .pbxproj settings. Testing indicates that
+            // PROVISIONING_PROFILE_SPECIFIER take precedence, if it's not empty.
+            if (provProfileUUID || provProfileName) {
+                if (!provProfileUUID) {
+                    provProfileUUID = "";
+                }
+
+                if (!provProfileName) {
+                    provProfileName = "";
+                }
+
+                xcode_provProfile = `PROVISIONING_PROFILE=${provProfileUUID}`;
+                xcode_provProfileSpecifier = `PROVISIONING_PROFILE_SPECIFIER=${provProfileName}`;
```

Comment:
For AppCenter reviewers: I tried putting double quotes around the name (or empty string), but given that we add the arguments individually, the double quotes make it to Xcode and Xcode fails because it can't find `"Bob"` or `""`.

Reply:
Sending empty like this in a node program is fine:

```provisioningProfileName: "",```

Avoid doing this (double quotes inside of single quotes):

```provisioningProfileName: '""',```